src/components/ContentSidebar/ActivityFeed/activity-feed/index.js

```diff
@@ -0,0 +1 @@
+export default from './ActivityFeed';
```

Comment:
```export { default } from ...;```
and
```export { default as Foo } from ...;```

is the current correct syntax

Reply:
Done. Out of curiosity, what's the reasoning behind this style?